File: AlgoExpert/1. Easy/Java/13. Palindrome Check/Palindrome Check [O(n^2)].java
class Program {

	// O(n^2) time | O(n) space
	public static boolean isPalindrome(String str) {
		String reversedString = "";
		for (int i = str.length() - 1; i >= 0; i--) {
			reversedString += str.charAt(i);
		}
		return str.equals(reversedString);
	}
}

File: AlgoExpert/1. Easy/C++/13. Palindrome Check/Palindrome Check [O(n^2)].cpp
#include <iostream>

using namespace std;

// O(n^2) time | O(n) space

bool isPalindrome(string str) {
	string reversedString = "";
	for (int i = str.length() - 1; i >= 0; i--) {
		reversedString += str[i];
	}
	return str == reversedString;
}